packages/orbit-components/src/InputField/index.tsx

```diff
@@ -463,7 +470,7 @@ const InputField = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
           {...dataAttrs}
         />
         {suffix && <Suffix>{suffix}</Suffix>}
-        <FakeInput disabled={disabled} error={error} />
+        <FakeInput readOnly={readOnly} disabled={disabled} error={error} />
```

Comment:
Read-only and disabled have different semantics, so I would prefer not to conflate them together under a prop named `disabled`.
It creates unnecessary coupling and would require a comment to explain why this is done like this (as it's rather surprising).

Reply:
Okay. It can be like that, disabled can be renamed to something else then also, as it’s only for applying styles and now we passing two props for applying one thing that can  relate to single condition instead of multiple ones inside :) 

It’s up to you, still will be removed soon

Reply:
> It’s up to you, still will be removed soon

Exactly :)